src/installer/tests/Microsoft.NET.HostModel.Tests/AppHost.Bundle.Tests/BundleExtractToSpecificPath.cs

```diff
@@ -57,15 +57,23 @@ private void Bundle_Extraction_To_Specific_Path_Succeeds()
             extractDir.Should().NotHaveFiles(BundleHelper.GetFilesNeverExtracted(fixture));
         }
 
-        [InlineData("./foo")]
-        [InlineData("../foo")]
-        [InlineData("foo")]
-        [InlineData("foo/bar")]
+        [InlineData("./foo", BundleOptions.None)]
+        [InlineData("../foo", BundleOptions.None)]
+        [InlineData("foo", BundleOptions.None)]
+        [InlineData("foo/bar", BundleOptions.None)]
+        [InlineData("./foo", BundleOptions.BundleNativeBinaries)]
+        [InlineData("../foo", BundleOptions.BundleNativeBinaries)]
+        [InlineData("foo", BundleOptions.BundleNativeBinaries)]
+        [InlineData("foo/bar", BundleOptions.BundleNativeBinaries)]
+        [InlineData("./foo", BundleOptions.BundleSymbolFiles)]
+        [InlineData("../foo", BundleOptions.BundleSymbolFiles)]
+        [InlineData("foo", BundleOptions.BundleSymbolFiles)]
+        [InlineData("foo/bar", BundleOptions.BundleSymbolFiles)]
         [Theory]
-        private void Bundle_Extraction_To_Relative_Path_Succeeds (string relativePath)
+        private void Bundle_Extraction_To_Relative_Path_Succeeds (string relativePath, BundleOptions bundleOptions)
         {
             var fixture = sharedTestState.TestFixture.Copy();
-            var singleFile = BundleSelfContainedApp(fixture, BundleOptions.None);
+            var singleFile = BundleSelfContainedApp(fixture, bundleOptions);
```

Comment:
Would it be too complicated to actually check that an expected file was extracted to the correct location for the non-`BundleOptions.None` cases?